db/structure.sql

```diff
@@ -2776,7 +2776,8 @@ CREATE TABLE posts (
     pixiv_id integer,
     last_commented_at timestamp without time zone,
     has_active_children boolean DEFAULT false,
-    bit_flags bigint DEFAULT 0 NOT NULL
+    bit_flags bigint DEFAULT 0 NOT NULL,
+    tag_count_meta integer DEFAULT 0 NOT NULL
```

Comment:
This may need an index but let's see how the queries work out in practice first.

Reply:
I didn't see an index on the other tag category counts, so I'm guessing that there's no issue...?